packages/paste-core/components/combobox/src/types.ts

```diff
@@ -37,21 +37,27 @@ export type HighlightedIndexChanges = {
 export interface ComboboxProps extends Omit<InputProps, 'id' | 'type' | 'value'>, Pick<BoxProps, 'element'> {
   autocomplete?: boolean;
   helpText?: string | React.ReactNode;
+  labelText: string | NonNullable<React.ReactNode>;
+  optionTemplate?: OptionTemplateFn<any>;
+  groupLabelTemplate?: (groupName: string) => React.ReactNode;
+  groupItemsBy?: string;
+  variant?: InputVariants;
+
+  // Downshift useCombobox Hook Props
```

Comment:
suggestion (non-blocking): it took me a while to find the A11y props in the downshift documentation bc it isn't actually in their site, it's on the ReadMe for their repo. Might be good to paste the link to that here for future reference.

Reply:
That's not a bad call, they're pretty hard to find.